core/src/main/scala/org/genivi/sota/core/DeviceUpdatesResource.scala

```diff
@@ -65,9 +66,15 @@ class DeviceUpdatesResource(db: Database,
 
   protected lazy val updateService = new UpdateService(DefaultUpdateNotifier, deviceRegistry)
 
+  private def sendDeviceSeen(id: Uuid): Future[Unit] = {
+    async {
+      val namespace = await(deviceRegistry.fetchMyDevice(id).map(_.namespace))
```

Comment:
I spoke to Jerry about this and he said that we should just do it this way for now, this will be fazed out eventually anyway in favour of having the device gateway send the message, which does also have the namespace.